docs/bis/guide/fundamentals/attribution-guidelines.md

```diff
@@ -0,0 +1,71 @@
+# Attribution: Guidelines
```

Comment:
When I read the title of this page I thought 'assigning cause or credit' rather than 'defining a quality or characteristic'.  I know it can mean both but I associate it with the first definition more than the second.  Do we have a better way to describe this?

Reply:
Getting inspiration from the title of this PR... what about "Modeling data as Attribution: Guidelines"

Reply:
'attribution' feels out of place in the context of the rest of the BIS docs because we generally talk about 'properties' instead of 'attributes'.  Searching the existing docs I only saw one or two instances where we used attributes for business properties.  All other uses of attribute were in EC for schema/schema items or Custom Attributes

Reply:
I went with 'attribution' here because the article covers "Json data" - so at that point the term "property" felt more like the implementation of an 'attribution' in terms of either a first-class or an aspect ECProperty.

If we keep that mindset (abstract: attribution, implementation: property), then I should probably revise the entire guide to align to it.

Reply:
Attribution still doesn't seem like the right term even if we want to use 'attribute' over 'property'

Reply:
JSON Schemas used the term 'properties' similar to ECSchema: https://json-schema.org/learn/getting-started-step-by-step#define-properties